### DISCUSSION TITLE
[Unknown Mod][Adventure Map] Adjust the text in the Witch's Hut window when the hero does not already have the proposed secondary skill and there is no room for another secondary skill

### DISCUSSION THREAD

**Discussion by LeHerosInconnu** [ihhub/fheroes2#4695] [Ideas]
Adjust the text in the Witch's Hut window when the hero does not already have the proposed secondary skill and there is no room for another secondary skill.

This:

"You already illuminate the firmament of learning!"

Instead of:

"You already know everything you deserve to learn!"

In fheroes2:

![Witch's Hut max skills 001](https://user-images.githubusercontent.com/43583994/143456162-7f3bbf6e-a157-42bd-aa68-fa48e823afdd.png)

![Witch's Hut max skills 002](https://user-images.githubusercontent.com/43583994/143456197-269bad2c-ff8f-4f07-96ca-03d06d0807c9.png)

Here is the save file:
[Before Witch Hut max skills 0.zip](https://github.com/ihhub/fheroes2/files/7603576/Before.Witch.Hut.max.skills.0.zip)

This is not an option.
This can be implemented directly in fheroes2.

**Comment by a1exsh** [comment]
> no more free room

Free room where — in hero's head? 😂 

I think the current message is pretty neat, as it makes the game more lively.

**Reply by LeHerosInconnu** [reply]
Hello @a1exsh,

> > no more free room
> 
> Free room where — in hero's head? :joy:
> 
> I think the current message is pretty neat, as it makes the game more lively.

I used the word "room" because it is also used for troops and artifacts if I am not mistaken.
But yes, it's true that it looks a bit weird. :D
Maybe something like that:

"You are too foolish to learn anything else!"

"Your head is ready to explode with all the skills you have learned, one more and it's a splash out!"

"You small brain, you crazy!"
